community-website/src/community-project-boilerplate-docgen/src/widgets/instantsearch.md

```diff
@@ -60,10 +60,14 @@ export class AppComponent {}
 > A hook that will be called each time a search needs to be done, with the helper as a parameter. It’s your responsibility to call `helper.search()`. This option allows you to avoid doing searches at page load for example.
 
 `createAlgoliaClient?: (algoliasearch: Function, appId: string, apiKey: string) => CustomClient`
+> _Deprecated in favor of `searchClient`._
```

Comment:
We do not want to deprecate this yet, it is still used for the server side rendering strategy: 

```js
import {
  createSSRAlgoliaClient,
  parseServerRequest
} from "angular-instantsearch";

[...]

constructor(
  private httpClient: HttpClient,
  private transferState: TransferState,
  private injector: Injector,
  @Inject(PLATFORM_ID) private platformId: Object
) {
  const req = isPlatformServer(this.platformId)
    ? this.injector.get("request")
    : undefined;

  const searchParameters = parseServerRequest(req);

  this.instantSearchConfig = {
    searchParameters,
    appId: "latency",
    apiKey: "6be0576ff61c053d5f9a3225e2a90f76",
    indexName: "ikea",
    urlSync: true,
    createAlgoliaClient: createSSRAlgoliaClient({
      makeStateKey,
      HttpHeaders,
      transferState: this.transferState,
      httpClient: this.httpClient
    })
  };
}
```

If we want to get rid of `createAlgoliaClient` we have to support server side rendering as well with the new method.

I can help for this 👍

Reply:
I added `createSSRSearchClient`. Hope this is how you envisioned it!